andela/k2-cfh
# #153009250 integrate HoundCI with badge

#### What does this PR do?

Integrate HoundCI for automated project codebase review
 
#### Description of Task to be completed?

- Signin with Github on http://houndci.com
- See list of repositories on the account
- Activate the k2-cfh repository

#### How should this be manually tested?

- Make changes to javascript codes on project codebase
- Create a pull request
- Check hound review on pull request

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

Story ID - 153009250

#### Screenshots (if appropriate)

<img width="1174" alt="screen shot 2017-11-20 at 4 11 05 pm" src="https://user-images.githubusercontent.com/32123313/33026753-70615ff6-ce11-11e7-9288-6d73dbbd6baa.png">

#### Questions:

N/A
